src/components/Carousel.tsx

```diff
@@ -0,0 +1,156 @@
+import { Div, DivProps, Flex } from 'honorable'
+import { Children, useEffect, useState } from 'react'
+import { CSSTransition } from 'react-transition-group'
+import { keyframes } from '@emotion/react'
+
+export type CarouselProps = DivProps & {
+  autoAdvanceTime?: number
+} 
+
+const dotAnimationIn = keyframes`
+0% {
+  transform: scale(1,1)
+}
+50% {
+  transform: scale(1.15, 1.15)
+}
+100% {
+  transform: scale(1, 1)
+}
+`
+
+function Dot({ active = false, ...props }: DivProps & { active?: boolean }) {
+  return (
+    <Div
+      padding="4px"
+      cursor="pointer"
+      {...props}
+    >
+      <Div
+        backgroundColor={active ? 'action-link-inline' : 'fill-two'}
+        transition="background-color 0.35s cubic-bezier(.20,.55,.80,.45)"
+        animationName={active ? dotAnimationIn : null}
+        animationDuration="0.75s"
+        animationIterationCount="1"
+        width="8px"
+        height="8px"
```

Comment:
width={8}
height={8}

Reply:
Do you think it's a little confusing to have unitless values refer to pixels for width/height but to multiples of a "space" for padding/margin/etc?